.docker/Pipfile

```diff
@@ -22,6 +22,7 @@ nbval = "*"
 nest-asyncio = "<=1.5.4"
 nltk = ">=3.6.4"
 nncf = {extras = ["torch", "tf"]}
+onnx = "==1.11.0"
```

Comment:
This may impact another notebook. Could you please change this to ">=1.11.0"

Reply:
Fixed, thx.

Reply:
Please help check if onnx 1.11.0 has conflict with other demo, otherwise I can only go back to use workaround by inserting "import onnx" in source to overcome the issue on windows. It is onnx issue actually.

Reply:
It seems onnx 1.11.0 remains not compatible issue during docker build validation. I remove onnx version specification, and use "import onnx" workaround in source to overcome.